pkg/cmd/codespace/create.go

```diff
@@ -119,12 +120,22 @@ func (a *App) Create(ctx context.Context, opts createOptions) error {
 
 	promptForRepoAndBranch := userInputs.Repository == ""
 	if promptForRepoAndBranch {
+		var defaultRepo string
+		remotes, _ := a.remotes()
+		if remotes != nil {
+			defaultRemote, _ := remotes.ResolvedRemote()
+			if defaultRemote != nil {
+				defaultRepo = ghrepo.FullName(defaultRemote)
+			}
+		}
```

Comment:
Small style nit:
```suggestion
		if remotes, err := a.remotes(); err != nil {
			if defaultRemote, err := remotes.ResolvedRemote(); err != nil {
				defaultRepo = ghrepo.FullName(defaultRemote)
			}
		}
```

Reply:
We're ignoring the errors deliberately here, so I think you meant `if remotes, _ := a.remotes(); remotes != nil {` right (and the same for the second line)? I've made that change.